e2e/tests/settingsConfig.spec.ts

```
@@ -17,112 +17,228 @@ test.describe('Settings Tests', () => {
+    test(`@admin Modify maxInfoQueries via UI validation via API: Input value: 999`, async () => {
+        const settingsConfigPage = await browser.createNewPage(SettingsConfigPage, urls.settingsUrl)
+        await settingsConfigPage.modifyRoleValue(roles.maxInfoQueries, "999")
+        await settingsConfigPage.refreshPage();
+        await settingsConfigPage.scrollToBottomInTable();
+        const res = await settingsConfigPage.getRoleContentValue(roles.maxInfoQueries);
+        console.log("ui value: ", res);
+        
+        await new Promise(resolve => { setTimeout(resolve, 3000) });
+        const apiCall = new ApiCalls()
+        let value;
+        for (let i = 0; i < 5; i++) {
```

ESLint

[error] Unary operator '++' used.

(no-plusplus)